src/queries/community-champions.js

```diff
@@ -0,0 +1,30 @@
+export const communityChampions = `
+    query CommunityChampions {
+        allStrapiCommunityChampions(
+            sort: { fields: [firstName, middleName, lastName] }
```

Comment:
Nice! Do we plan on re-using this query later? If not, we probably don't need to sort if this is just for creating the pages